docs/build-blockchain/chains/v2/ethereum/about-ethereum.md

```diff
@@ -17,37 +26,16 @@ The Ethereum network is one of the most popular platforms for building dApps and
 
 [**Github**](https://github.com/ethereum/eth1.0-apis)
 
-
 ---
 
 ## Connect wallet
 
-You can set up your **MetaMask wallet** to connect to Ethereum RPC. You can then perform transactions and interact with the network.
-
-### Get started
-
-1. Open your **Metamask Extension** and click the '_**Network**_' drop down menu at the top.&#x20;
-2. Select '_**Custom RPC**_'.&#x20;
-3. Enter the settings for the required project as follows in the table below:
-
-| **Chain** | **Custom RPC Category** | **Details**               |
-| --------- | ----------------------- | ------------------------- |
-| Ethereum  | NETWORK NAME:           | Ethereum RPC              |
-|           | NEW RPC URL:            | https://rpc.ankr.com/eth/ |
-|           | CHAIN ID:               | 1                         |
-|           | SYMBOL:                 | ETH                       |
-|           | BLOCK EXPLORER URL:     | https://etherscan.io/     |
-
+You can set up your **MetaMask wallet** to connect to Ethereum RPC. You can then perform transactions and interact with the network. 
 
-## Integrate code
 
-### web3 library
+#### Example request - web3_clientVersion
 
-- **clientVersion**
-
-Returns the current client version
-
-#### Example request
+This request gets the Client version being used. 
```

Comment:
This is just a suggestion - it might be better to use `eth_blockNumber` instead of `web3_clientVersion` because it's a bit more relatable and useful.

Reply:
Agreed. I'll publish this and then add it as a task to update.